easybuild/easyconfigs/a/awscli/awscli-2.1.37-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,70 @@
+easyblock = 'PythonBundle'
+
+name = 'awscli'
+version = '2.1.37'
+
+homepage = 'https://pypi.python.org/pypi/awscli'
+description = 'Universal Command Line Environment for AWS'
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+builddependencies = [
+    ('binutils', '2.35'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('PyYAML', '5.3.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('jmespath', '0.10.0', {
+        'checksums': ['b85d0567b8666149a93172712e68920734333c0ce7e89b78b3e987f71e5ed4f9'],
+    }),
+    ('botocore', '1.20.53', {
+        'source_tmpl': '2c5e5782c4.tar.gz',
+        'source_urls': ['https://github.com/boto/botocore/archive/'],
```

Comment:
Does this need to be different from the other easyconfig?

Reply:
different? swapping the order of tmpl and URLs?  not sure why I did that

Reply:
No, compare with botocore in awscli 1.19.53

Reply:
Yes, botocore does not support AWScli 2 yet. Needs to be pulled from the archive.